manifests/diff/diff.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -euo pipefail
+
+# Function to generate diff output with individual file toggles
+generate_diff_output() {
+    local pattern=$1
+    local output=""
+
+    # First find all directories matching the pattern, then find files in them
+    while IFS= read -r -d '' dir; do
+        while IFS= read -r -d '' file; do
+            local diff_result
+            diff_result=$(skipctl manifests diff -p "$file" --ref "${INPUTS_REF}" --diff-format patch 2>/dev/null || true)
+
+            if [ -n "$diff_result" ]; then
+                local file_path="$file"
+                # Remove the input path prefix for cleaner display
+                local display_path="${file_path#${INPUTS_PATH}/}"
+
+                output+="<details>"
+                output+=$'\n'
+                output+="  <summary><b>$display_path DIFF</b></summary>"
+                output+=$'\n'
+                output+=$'\n'
+                output+='```diff'
+                output+=$'\n'
+                output+="$diff_result"
+                output+=$'\n'
+                output+='```'
+                output+=$'\n'
+                output+="</details>"
+                output+=$'\n'
+            fi
+        done < <(find "$dir" -type f -print0 | sort -z)
+    done < <(find "${INPUTS_PATH}" -type d -name "$pattern" -print0 | sort -z)
```

Comment:
This is for advanced users. It's working, but not for everyone to understand this 😅 

Would add a meme, but:

<img width="1153" height="861" alt="Image" src="https://github.com/user-attachments/assets/dc1143d1-06ae-4b50-8cdc-a78c1ab62665" /> `

Reply:
Leave it as-is for now, but we could consider doing this in e.g., Python for readability.